<ui version="4.0" >
 <class>QSPageSettingsClass</class>
 <widget class="QWidget" name="QSPageSettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string> Page Settings</string>
  </property>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>Margins in millimeters</string>
   </property>
   <widget class="QWidget" name="" >
    <property name="geometry" >
     <rect>
      <x>12</x>
      <y>25</y>
      <width>151</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="2" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>(mm)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Left</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QSpinBox" name="spinBox_2" />
     </item>
     <item row="3" column="2" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>(mm)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QSpinBox" name="spinBox_3" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Bottom</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Right</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>(mm)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="spinBox" />
     </item>
     <item row="1" column="2" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>(mm)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QSpinBox" name="spinBox_4" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Top</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
